packages/stream_chat_persistence/README.md

```
@@ -1,58 +1,65 @@
 - [Register](https://getstream.io/chat/trial/) to get an API key for Stream Chat
-- [Flutter Chat Tutorial](https://getstream.io/chat/flutter/tutorial/) 
-- [Chat UI Kit](https://getstream.io/chat/ui-kit/)
-- [Flutter Offline Docs](https://getstream.io/chat/docs/flutter-dart/flutter_offline/)
-
-This package provides a persistence client for fetching and saving chat data locally.
-Stream Chat Persistence uses [Drift](https://github.com/simolus3/drift) as a disk cache.
+- [Flutter Chat Tutorial](https://getstream.io/chat/flutter/tutorial/)
+- [Documentation](https://getstream.io/chat/docs/sdk/flutter/)
+- [V10 Migration Guide](https://getstream.io/chat/docs/sdk/flutter/guides/migration-guide-10-0/)
 
 ### Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)